app/components/peopletable/peopletable.js

```diff
@@ -299,10 +198,54 @@ export const PeopleTable = translate()(class PeopleTable extends React.Component
   handleOverlayClick() {
     this.setState({
       showModalOverlay: false,
-      currentRowIndex: -1,
     });
   }
 
+  handleClickPwD(link) {
+    return () => {
+      this.props.trackMetric('Selected PwD');
+      this.props.push(link);
+    }
+  }
+
+  handleClickEdit(patient) {
+    return () => {
+      this.props.trackMetric('Clicked Edit PwD');
+      this.props.push(`/patients/${patient.userid}/profile#edit`);
+    }
+  }
+
+  handleSearchChange(event) {
+    this.setState({search: event.target.value});
+  }
+
+  renderPatient = ({fullName, email, link}) => {
+    return (
+      <Flex alignItems="center" onClick={this.handleClickPwD(link)} sx={{cursor: 'pointer', height: '2.8em'}}>
+        <Box>
+          <Text fontWeight="medium">{fullName}</Text>
+          <Text>{email}</Text>
+        </Box>
+      </Flex>
+    );
+  }
```

Comment:
Without a hard-coded height, the cell height is different for users with and without an email address, which wouldn't seem to be an issue until you page through results and the `Next` button jumps around underneath the cursor and prevents you from just repeat clicking to page through results.

Reply:
Ah.  Would `min-height` be better (in case the name wraps at smaller breakpoints)? Or perhaps an empty space in lieu of an empty email address so that it always renders a second line?

I was thinking that patients would automatically have an email address - I suppose there are cases where they don't?

Reply:
It's strongly encouraged (in multiple places now) for a clinician to add an email address, but it's not strictly required in the clinic account creation workflow. `min-height` would work similarly. Having an empty space felt (just barely) like more of a work-around than a set height, though I'm not totally thrilled with either so I'm totally open to the suggestions. (I'd be much happier if the `lh` unit were broadly available and just use `2lh`, but alas the future is always just out of reach)

Reply:
I'm good with whatever option you go with here (though I do _slightly_ prefer `min-height` to `height`)

Reply:
Going with the unicode non-breaking space (since React is happier about escaping with /u strings).